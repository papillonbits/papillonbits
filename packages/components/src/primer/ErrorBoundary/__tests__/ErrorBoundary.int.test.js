import { regular } from './ErrorBoundary.int.story'

describe('<ErrorBoundary />', () => {
  describe('Render', () => {
    test('must match regular()', () => {
      expect(global.renderToJSON(regular())).toMatchSnapshot()
    })
  })
})
