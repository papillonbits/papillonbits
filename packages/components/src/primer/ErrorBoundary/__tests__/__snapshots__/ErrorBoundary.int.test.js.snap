// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorBoundary /> Render must match regular() 1`] = `<div />`;
